Sources/System/WindowsPlatformConstants.swift

```diff
@@ -0,0 +1,330 @@
+/*
+ This source file is part of the Swift System open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift System project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+*/
+
+#if os(Windows)
+
+import ucrt
```

Comment:
What is this import used for in this file? We also want to make sure that System doesn't pull in any other modules, so make sure all imports from the System module are `@_implementationOnly`

Reply:
Leftovers :-( I realized that later that the values are being replicated rather than imported.